Formula/otf2.rb

```diff
@@ -0,0 +1,27 @@
+class Otf2 < Formula
+  desc "Open Trace Format 2 file handling library"
+  homepage "https://www.vi-hps.org/projects/score-p/"
+  url "https://www.vi-hps.org/cms/upload/packages/otf2/otf2-2.1.1.tar.gz"
+  sha256 "01591b42e76f396869ffc84672f4eaa90ee8ec2a8939755d9c0b5b8ecdcf47d3"
+
+  depends_on "python"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    system "make"
+    system "make", "install"
+  end
+
+  test do
+    cp_r "#{share}/doc/otf2/examples", testpath
+    chdir "#{testpath}/examples" do
+      # build serial tests
+      system "make", "serial"
+      system "./otf2_writer_example"
+      system "./otf2_reader_example"
```

Comment:
It would be good to verify output from running commands. I guess if the reader reads the example that the writer wrote this might be ok. But it would be good to test for the creation of expected files and/or expected strings in stdout indicating success.

Reply:
@XL64 Any chance you could update the tests with additional checks?

Reply:
Let's also add the MPI checks too, and Fortran.